ramp-engine/ramp_engine/aws/api.py

```diff
@@ -251,26 +251,28 @@ def launch_ec2_instances(config, nb=1):
 def _get_image_id(config, image_name):
     sess = _get_boto_session(config)
     client = sess.client('ec2')
+
+    # get all the images with the given imange_name in the name
     result = client.describe_images(Filters=[
         {
             'Name': 'name',
-            'Values': [
-                image_name
-            ]
+            'Values': [image_name+'*'
+                       ],
         }
     ])
+
     images = result['Images']
     if len(images) == 0:
         raise ValueError(
             'No image corresponding to the name "{}"'.format(image_name))
     elif len(images) > 1:
-        raise ValueError(
-            'Multiple images corresponding to the name "{}".'
-            ' Please fix that'.format(image_name))
+        # get only the newest image if there are more than one
+        image = sorted(images, key=lambda x: x['CreationDate'],
+                       reverse=True)[0]
     else:
         image = images[0]
```

Comment:
you could actually remove the `elif` and the `else`.
It will work so let's not complexify the code here.